Source/Core/AudioCommon/PulseAudioStream.h

```diff
@@ -20,7 +20,7 @@ class PulseAudio final : public SoundStream
   ~PulseAudio() override;
 
   bool Init() override;
-  bool SetRunning(bool running) override { return running; }
+  bool SetRunning(bool running) override { return true; }
```

Comment:
Should we add a comment to SoundStream::SetRunning to clarify what SetRunning is supposed to return (whether the operation succeeded, rather than whether the stream is now running)?

Reply:
meh^^